README.md

```diff
@@ -20,13 +20,98 @@ If you prefer a `<script>` tag, you can get it from `window.PropTypes` global:
 ```html
 <!-- development version -->
 <script src="https://unpkg.com/prop-types/prop-types.js"></script>
- 
+
 <!-- production version -->
 <script src="https://unpkg.com/prop-types/prop-types.min.js"></script>
 ```
 
 ## Usage
 
+Here is an example documenting the different validators provided:
+
+```javascript
+import PropTypes from 'prop-types';
+
```

Comment:
That makes sense - I'll rewrite and add to this so that it would make sense to someone who has not got the context of React.